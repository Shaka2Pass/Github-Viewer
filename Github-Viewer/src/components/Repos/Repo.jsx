import React from 'react';
import PropTypes from 'prop-types';

const Repo = ({ name, url }) => (
  <div>
    <div>
      Name: {name}
    </div>
    <div>
      Url: {url}
    </div>
  </div>
);

Repo.propTypes = {
  name: PropTypes.string.isRequired,
  url: PropTypes.string.isRequired, 
};

export default Repo;
