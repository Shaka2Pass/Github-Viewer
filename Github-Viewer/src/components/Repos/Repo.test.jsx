import React from 'react';
import { shallow } from 'enzyme';
import Repo from './Repo';

describe('Repo', () => {
  it('matches a user', () => {
    const wrapper = shallow(<Repo name="shaka2pass" url="https://github.com/Shaka2Pass/About-Me-Guessing-Game"/>);
    expect(wrapper).toMatchSnapshot();
  });
});
