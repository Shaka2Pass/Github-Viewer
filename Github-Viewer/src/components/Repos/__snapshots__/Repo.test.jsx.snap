// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Repo matches a user 1`] = `
<div>
  <div>
    Name: 
    shaka2pass
  </div>
  <div>
    Url: 
    https://github.com/Shaka2Pass/About-Me-Guessing-Game
  </div>
</div>
`;
